# Request
bool turn_direction
---
# Result
geometry_msgs/PoseStamped pose
---
# Feedback